deployment/docker-compose.latest.yml

```diff
@@ -0,0 +1,89 @@
+# Development Setup.
+# Uses the latest commit of the `master` branch to run each service.
+# Start individual services using the following command::
```

Comment:
> Additionally, a docker-compose.latest.yml has been added
as a drop-in replacement for docker-compose.override.yml,
which is a more explicit name for the file imo.

The `docker-compose.override.yml` allowed to run the local code including all changes that are not committed and pushed, yet. AFAICS, this does not work with `docker-compose.latest.yml`. That is only relevant for people developing the services, not for anyone who just wants to run the latest version from master.

It's just a nice to have, so I won't insist on having that functionality and it also does not work out of the box, as the keystore is not in the repo. But reusing the docker setup locally looked like a nice way to allow people to run services without digging out all the required CLI params.

If this is indeed meant to replace the `docker-compose.override.yml`, why didn't you remove it in the PR?

Reply:
>If this is indeed meant to replace the docker-compose.override.yml, why didn't you remove it in the PR?

Because I'm not sure if it isn't used for something else (which you confirmed just now). I assumed it was used to deploy the latest dev version to a server, not run them locally. 

If this is something you guys need/use, I'll gladly just leave it in there. However, if it does not "run out of the box", we should probably adapt it so it does, if that's possible.

Reply:
> However, if it does not "run out of the box", we should probably adapt it so it does, if that's possible.

I didn't want to put a walled with tokens inside the public repo, even if it's just test eth. That's the only thing missing for it to run. I don't see a good way to handle that. Do you see one?